samples-generator/specs/bin/CESIUM_3dtiles_batch_table_Spec.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+var minimalTriangleGLTF = require('./minimalTriangleGLTF');
+var create3dtilesBatchTableExt = require ('../../lib/create3dtilesBatchTableExt');
+
+describe('CESIUM_3dtiles_batch_table.js', function() {
```

Comment:
Style: drop the `.js`. Also use the name of the file instead of `CESIUM_3dtiles_batch_table`